gui/wxpython/psmap/utils.py

```diff
@@ -27,12 +27,9 @@
 import grass.script as gs
 from grass.exceptions import ScriptError
 
-try:
-    from PIL import Image as PILImage  # noqa: F401
```

Comment:
Can you make sure that other files needing PILImage are still working correctly? It seems some were importing from psmap.utils (not really good practice, but it's there now)